.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: "Bug Report"
```

Comment:
maybe we can add a `Severity` paramater for a bug report, which can help us focus.

Reply:
I think this may be better judged by the team members, so a github label might be more appropriate.

Reply:
I don't see how we would expose such a parameter to a user anyway